src/user/User.ts

```diff
@@ -35,4 +36,6 @@ User.init(
   }
 );
 
+User.hasMany(Token, { onDelete: 'cascade', foreignKey: 'userId' });
```

Comment:
Adding User-Token relationship, where User is going to have many Token with a foreignKey of 'userId'